ChangeLog.md

```diff
@@ -11,6 +11,11 @@ Changes that affect Michelson code are tagged with `[CONTRACT]`.
 Unreleased
 ==========
 
+0.8.0
+==========
+* [#172](https://github.com/tz-wrapped/tezos-btc/pull/172)
+  + Morley dependencies updated to provide changes that support Ithaca protocol.
+
```

Comment:
It looks like we neglected this changelog a bit, although most changes were dependencies and test updates.

I would only add a mention for #159:
```suggestion
* [#159](https://github.com/tz-wrapped/tezos-btc/pull/159)
  + Switch tests to nettest engine.

```
which seems a bit more note-worthy to me.